packages/snap-preact-components/src/components/Molecules/FacetGridOptions/readme.md

```diff
@@ -30,9 +30,9 @@ If using within Autocomplete, the `previewOnFocus` prop will invoke the `value.p
 
 ```jsx
 <Autocomplete>
-	...
```

Comment:
I think these dots are useful to show that this example is not complete, but just a snippet.